test/framework/options.py

```diff
@@ -6542,6 +6544,173 @@ def test_easystack_basic(self):
             regex = re.compile(pattern)
             self.assertTrue(regex.search(stdout), "Pattern '%s' should be found in: %s" % (regex.pattern, stdout))
 
```

Comment:
Also add test that uses incorrect format for EasyStack (i.e. with `-` before the EasyConfig name, so that `easyconfigs:` becomes a dict instead of a list). This format is invalid and won't be parsed correctly. It _should_ raise an error.

Reply:
Done in https://github.com/easybuilders/easybuild-framework/pull/4057/commits/cd8bb71c2432350943711c01b846f25217a81f0a